src/utils/BrowserCompatibilityModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+import bowser from "bowser";
+import { Modal } from "react-bootstrap";
+import { createPortal } from "react-dom";
+
+const SUPPORTED_BROWSERS = {
```

Comment:
Understood, I am just questioning whether those versions are ok even for Vite:
```  
 edge: ">=14",
  chrome: ">=48",
  chromium: ">=48",
  opera: ">=35",
  safari: ">=10.1",
  firefox: ">=44",
  ie: ">=11",
  samsung_internet: ">=6.2",
```

Because when I go to webpage https://vitejs.dev/guide/build#browser-compatibility, it shows me:
Chrome >=87
Firefox >=78
Safari >=14
Edge >=88

Reply:
You're right, those browser version listed in the link you've provided are valid, and it is written in docs "The production bundle assumes support for modern JavaScript. By default, Vite targets browsers which support the [native ES Modules](https://caniuse.com/es6-module), [native ESM dynamic import](https://caniuse.com/es6-module-dynamic-import), and [import.meta](https://caniuse.com/mdn-javascript_operators_import_meta):"

So I will change versions of the browsers as provided in the docs, but there are some other browsers such as opera, samsung_internet, chromium, that are not listed in the docs, and I'm not now sure what to do with them.
But at the same time we can use polyfills for the outdated browsers, as described here https://vitejs.dev/guide/build#browser-compatibility 

https://github.com/vitejs/vite/tree/main/packages/plugin-legacy

Reply:
I would:
- list only supported browsers from the documentation
- provide no polyfills (maybe we revise this decision later)